packages/eas-cli/src/build/build.ts

```diff
@@ -274,7 +279,24 @@ async function uploadProjectAsync<TPlatform extends Platform>(
             completedMessage: (duration: string) => `Uploaded to EAS ${chalk.dim(duration)}`,
           })
         );
-        return bucketKey;
+
+        const projectMetadataFile = await makeProjectMetadataFileAsync(projectTarball.path);
```

Comment:
Can we make it failsafe? Like, it's not crucial to run the build, so maybe if it fails it's ok and we should only log this error in Sentry or somewhere to know what should we fix?

Reply:
I can put it in `try`, but I don't see any way to log it to sentry in eas-cli

Reply:
Yeah, I think we don't have Sentry configured